pages/alerts.rst

```diff
@@ -284,11 +284,57 @@ Client URL
 .. image:: /images/alerts_pagerduty_notification.png
 
 
+.. _alert_notification_Slack Notification:
+
+Slack alert notification
+------------------------
+The Slack alert notification allows you to send notifications using your `Webhook URL <https://api.slack.com/messaging/webhooks>`__ to your slack workspace
+in response to the configured event definitions in your Graylog server.
+
+.. image:: /images/alerts_slack_notification.png
+
+These are the supported configuration options:
+
+Webhook URL
+    An unique URL to which you send a JSON payload with the message text and some options.
```

Comment:
I think this can just say "The unique URL used to send messages to your Slack instance."  Also, might want to move the URL link here instead of up in the description.